webapp/app/forms/steps/steuerlotse_step.py

```diff
@@ -67,6 +72,7 @@ def prepare_render_info(cls, stored_data, *args, **kwargs):
             next_url=None,
             submit_url=None,
             header_title=None,
+            plausible_data=plausible_data,
```

Comment:
Why define the method variable "plausible_data" just to assign it to the "plausible_data" of the RenderInfo?

Why not jus:
`plausible_data=get_plausible_data(cls.name, Config.PLAUSIBLE_DOMAIN),`
?